core/storage/activity/gae_models.py

```diff
@@ -54,3 +54,8 @@ def get_or_create(cls, list_name):
             entity.put()
 
         return entity
+
+    @classmethod
+    def export_data(cls, user_id): # pylint: disable=unused-argument
```

Comment:
Here and elsewhere: shouldn't we be following the format of get_deletion_policy(), which returns a constant? I don't think we should be defining an empty implementation; in general, implementations of Takeout should not be empty.

We should also enforce, in tests, that every storage model has a valid takeout policy. (Either it is NOT_APPLICABLE, or there is a function in that model class that actually implements takeout.)